test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -95,6 +95,11 @@ var (
 		gvr("extensions", "v1beta1", "deployments/rollback"): {"create": testDeploymentRollback},
 	}
 
+	// admissionExemptResources lists objects which are exempt from admission validation/mutation
+	admissionExemptResources = []schema.GroupVersionResource{
```

Comment:
a map[gvr]bool would make lookups simpler. also, add a comment here that in general, no other resources should be added to this exempted list